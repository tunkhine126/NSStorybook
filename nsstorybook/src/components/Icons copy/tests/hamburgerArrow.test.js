/**
 *
 * Tests for HamburgerArrow
 *
 * @see https://github.com/react-boilerplate/react-boilerplate/tree/master/docs/testing
 *
 */

import React from 'react';
import TestingComponent from 'components/shared/Testing';
import { render } from '@testing-library/react';
// import 'jest-dom/extend-expect'; // add some helpful assertions
import HamburgerArrow from '../hamburgerArrow';

describe('<HamburgerArrow />', () => {
  it('Expect to not log errors in console', () => {
    const spy = jest.spyOn(global.console, 'error');
    render(
      <TestingComponent>
        <HamburgerArrow sideNavOpen classes={{ openMenu: 'test' }} />
      </TestingComponent>
    );
    expect(spy).not.toHaveBeenCalled();
  });

  it('Should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <TestingComponent>
        <HamburgerArrow sideNavOpen={false} classes={{ openMenu: 'test' }} />
      </TestingComponent>
    );
    expect(firstChild).toMatchSnapshot();
  });
});
