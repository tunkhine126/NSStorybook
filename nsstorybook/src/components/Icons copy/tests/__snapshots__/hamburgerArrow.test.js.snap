// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HamburgerArrow /> Should render and match the snapshot 1`] = `
<svg
  class="test"
  fill="none"
  height="25"
  viewBox="4 0 22 10"
  width="25"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M7.5 9.5H18V8H7.5V9.5ZM7.5 5.75H18V4.25H7.5V5.75ZM7.5 0.5V2H18V0.5H7.5Z"
    fill="#9B9EA6"
    fill-rule="evenodd"
  />
  <path
    clip-rule="evenodd"
    d="M3.62197 8.12803L0.365025 4.87109L3.62197 1.61415L3.62197 8.12803Z"
    fill="#9B9EA6"
    fill-rule="evenodd"
    style="transform: translate(22px,0);"
  />
</svg>
`;
